config/initializers/bootstrap_errors.rb

```diff
@@ -0,0 +1,30 @@
+ActionView::Base.field_error_proc = Proc.new do |html_tag, instance|
+  html = %Q(<span class="field_with_errors">#{html_tag}</span>).html_safe
+
+  form_fields = [
+    'textarea',
+    'input',
+    'select'
+  ]
+
+  elements = Nokogiri::HTML::DocumentFragment.parse(html_tag).css form_fields.join(', ')
```

Comment:
Line is too long. [88/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.